lib/Reader/readerir.cpp

```diff
@@ -4236,44 +4267,55 @@ bool GenIR::fgOptRecurse(mdToken Token) {
   return true;
 }
 
-void GenIR::returnOpcode(IRNode *Opr, bool IsSynchronousMethod) {
+void GenIR::returnOpcode(IRNode *Opr, bool IsSynchronizedMethod) {
   const ABIArgInfo &ResultInfo = ABIMethodSig.getResultInfo();
   const CallArgType &ResultArgType = MethodSignature.getResultType();
   CorInfoType ResultCorType = ResultArgType.CorType;
+  Value *ResultValue = nullptr;
 
   if (ResultCorType == CORINFO_TYPE_VOID) {
```

Comment:
Nit: the condition could be lifted and cached in a local (`bool IsVoidReturn = ResultCorType == CORINFO_TYPE_VOID;`)